hedera-node/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -109,6 +117,43 @@ public static Account createFromGrpc(final Id accountId, final CryptoCreateTrans
 		return created;
 	}
 
+	/**
+	 * Updates an existing {@link Account} instance from the given gRPC transaction body.
+	 * Note: The created model is not added to state, and must be explicitly persisted via {@link com.hedera.services.store.AccountStore#persistAccount(Account)}
+	 *
+	 * @param newKey
+	 * 		account key
+	 * @param newMemo
+	 * 		account memo
+	 * @param newAutoRenewPeriod
+	 * 		account auto renew period
+	 * @param newExpirationTime
+	 * 		account expiry
+	 * @param newProxyAccountID
+	 * 		account proxy ID
+	 * @param newReceiverSigRequiredWrapper
+	 * 		wrapped boolean condition receiver signature required
+	 * @return the updated account model
+	 */
+	public Account updateFromGrpc(Optional<Key> newKey,
+								  Optional<StringValue> newMemo,
+								  Optional<Duration> newAutoRenewPeriod,
+								  Optional<Timestamp> newExpirationTime,
+								  Optional<AccountID> newProxyAccountID,
+								  Optional<BoolValue> newReceiverSigRequiredWrapper,
+								  Optional<Int32Value> newMaxAutomaticTokenAssociations) {
```

Comment:
Nit: Instead of using the gRPC's version of optional (e.g. the wrappers - *Value ), I think it's safe to use the actual values wrapped in java optionals

Reply:
This can also simplify the unit tests

Reply:
Agree, will use the actual values wrapped in java optionals